docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx

```diff
@@ -504,20 +504,23 @@ clusterMetadata:
       rpcAddress: "127.0.0.1:8233"
 ```
 
-Then you can use the `tctl admin` tool to add cluster connections. All operations should be executed in both Clusters.
+Then you can use the Temporal CLI tool to add cluster connections. All operations should be executed in both Clusters.
 
+<!--tctl -address 127.0.0.1:7233 admin cluster upsert-remote-cluster --frontend_address "localhost:8233"-->
+<!--tctl -address 127.0.0.1:8233 admin cluster upsert-remote-cluster --frontend_address "localhost:7233"-->
+<!--tctl -address 127.0.0.1:7233 admin cluster upsert-remote-cluster --frontend_address "localhost:8233" --enable_connection false
+tctl -address 127.0.0.1:8233 admin cluster upsert-remote-cluster --frontend_address "localhost:7233" --enable_connection false-->
+<!--tctl -address 127.0.0.1:7233 admin cluster remove-remote-cluster --cluster "clusterB"
+tctl -address 127.0.0.1:8233 admin cluster remove-remote-cluster --cluster "clusterA"-->
+
+<!--THIS MUST BE CHECKED FOR ACCURACY-->
 ```shell
-# Add cluster B connection into cluster A
-tctl -address 127.0.0.1:7233 admin cluster upsert-remote-cluster --frontend_address "localhost:8233"
-# Add cluster A connection into cluster B
-tctl -address 127.0.0.1:8233 admin cluster upsert-remote-cluster --frontend_address "localhost:7233"
+# Add a cluster
+temporal operator cluster upsert --frontend_address="127.0.2.1:8233"
 
 # Disable connections
-tctl -address 127.0.0.1:7233 admin cluster upsert-remote-cluster --frontend_address "localhost:8233" --enable_connection false
-tctl -address 127.0.0.1:8233 admin cluster upsert-remote-cluster --frontend_address "localhost:7233" --enable_connection false
+temporal operator cluster upsert --frontend_address="localhost:8233" --enable_connection false
 
 # Delete connections
-tctl -address 127.0.0.1:7233 admin cluster remove-remote-cluster --cluster "clusterB"
-tctl -address 127.0.0.1:8233 admin cluster remove-remote-cluster --cluster "clusterA"
+temporal operator cluster remove --name="someClusterName"
```

Comment:
@cretz Need your help with this section, too. AFAIK:

- `--enable_connection` is now `--enable-connection`
- `--enable_connection false` needs to be `--enable-connection=false`

I was not able to manually test this, however, since I don't know how to set up a clustered server (wonder if this is something `server start-dev` should support)?

Reply:
> `--enable_connection` is now `--enable-connection`

Correct, done in #465

> `--enable_connection false` needs to be `--enable-connection=false`

I believe this was always defaulted to false and therefore should never have been set like this in the guide